src/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import debounce from 'lodash.debounce';
+import cn from 'classnames';
+
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  selectedPerson: Person | null;
```

Comment:
It is up to you to decide - but if I needed to save the selected person, I would do it inside `DropDown` component